playbook/app/pb_kits/playbook/pb_selectable_card/selectable_card.rb

```diff
@@ -72,6 +78,10 @@ def checked_class
       def enable_disabled_class
         disabled ? "disabled" : "enabled"
       end
+
+      def error_class
+        error ? " error" : ""
```

Comment:
Then you don't need the space before " error" here.

Reply:
ok I just copy pasted that, so ill fix it